fpdf/html.py

```diff
@@ -439,6 +439,14 @@ def output_table_sep(self):
     def handle_starttag(self, tag, attrs):
         attrs = dict(attrs)
         LOGGER.debug("STARTTAG %s %s", tag, attrs)
+        # if tag == "dl":
+        #     self.pdf.ln()
+        if tag == "dt":
+            self.pdf.ln(self.h + 2)
```

Comment:
Why `+ 2` here and 2 lines below?

Reply:
Yes, I misunderstood it and fixed at 70ae92ecec516326659165398b12769838f072d3.
The `dt` elements should be displayed In bold style now.

<img width="280" alt="Screen Shot 2022-09-17 at 12 54 07 AM" src="https://user-images.githubusercontent.com/69574727/190846658-08e182ed-66f9-48a2-8a7e-4b5073791aae.png">